lib/widgets/product_listing.dart

```diff
@@ -517,3 +522,22 @@ class _ProductListingState extends State<ProductListing> {
     }
   }
 }
+
+  StatelessWidget _getProductStockQuantityTag(
```

Comment:
We’re following a convention where all the return objects whose base type is `Widget` will return `Widget`. This will help us to make changes to the return value (for example, enclose a widget in a `Container` - whose base type is `Widget`) etc. So can you please make this `Widget`, like so:

`Widget _getProductStockQuantityTag(`